api/app/controllers/spree/api/base_controller.rb

```diff
@@ -122,19 +138,26 @@ def find_product(id)
       end
 
       def product_scope
+        variants_associations = [{ option_values: :option_type }, :default_price, :prices, :images]
         if current_api_user.has_spree_role?("admin")
           scope = Product.with_deleted.accessible_by(current_ability, :read)
+            .includes(:product_properties, :option_types, variants_including_master: variants_associations)
```

Comment:
Line is too long. [107/80]<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.